utils.py

```diff
@@ -0,0 +1,18 @@
+import os
+import yaml
+from categories.models.category import Category
+
+
+def get_base_category():
+    """
+    Function to get the category instance with slug
```

Comment:
Documentation of function should be imperative. Examples:
- Get X
- Calculate Y
- Do Z